src/types/types.ts

```diff
@@ -412,6 +412,14 @@ export interface Media {
   size?: number;
 }
 
+/** Detailed information on whether and why an NFT contract was classified as spam. */
+export interface SpamInfo {
+  isSpam: boolean;
+
+  /** A list of reasons why an NFT contract was marked as spam. */
+  classifications: string[];
```

Comment:
Can we turn this into a string enum with all the possible reasons? ex: https://github.com/alchemyplatform/alchemy-sdk-js/blob/347f87ac63f6fdc4d72683bf458511fe208c14fe/src/types/types.ts#L703

Reply:
done!